src/ansys/dyna/core/keywords/keyword_classes/auto_keywords.py

```diff
@@ -996,6 +996,7 @@
 from ansys.dyna.core.keywords.keyword_classes.auto.control_thermal_nonlinear import ControlThermalNonlinear
 from ansys.dyna.core.keywords.keyword_classes.auto.control_thermal_solver import ControlThermalSolver
 from ansys.dyna.core.keywords.keyword_classes.auto.control_thermal_timestep import ControlThermalTimestep
+from ansys.dyna.core.keywords.keyword_classes.auto.control_timestep import ControlTimestep
 from ansys.dyna.core.keywords.keyword_classes.auto.control_time_step import ControlTimeStep
```

Comment:
This will lead to problems... Your new import is being overwritten by the next import since you are importing the same object (namewise)

Reply:
@RobPasMue Hi Roberto, sorry for bothering you. I figured the naming convention for the issues. Regarding your comment. There are two classes that are imported:
ControlTimestep  (with lower S)
ControlTimeStep (with capital S)

Reply:
Ah my bad... I didn't read properly! Then all good.